db/primary_migrate/20230316155132_add_aal_string_value_to_identity.rb

```diff
@@ -0,0 +1,5 @@
+class AddAalStringValueToIdentity < ActiveRecord::Migration[7.0]
+  def change
+    add_column :identities, :aal_value, :text
+  end
+end
```

Comment:
I think we can remove this migration and only use the one below to add the column